src/components/all.scss

```diff
@@ -65,12 +68,32 @@ body {
       &:focus {
         opacity: 1;
       }
+
       svg,
       svg path {
         fill: $text-dark;
         stroke: $text-dark;
       }
     }
+    &.is-third {
```

Comment:
Just curiosity: What are the intentions with this CSS class?

Reply:
It is for this button. 
http://prntscr.com/svucnc

Reply:
Okay. I think you can just use this Button component: https://github.com/SEALAB-Software/international-website/blob/master/src/components/Button/Button.js

And specify `className="is-transparent` in the props. You will get the same result.

Reply:
> Fixed in the commit 
4090e6f